### ISSUE TITLE
Pygame plays sound at lower pitch than normal

### ISSUE THREAD

**Issue by ghost** [pygame/pygame#336]
I am trying to play one of the jingles from https://opengameart.org/content/8-bit-sound-effects-library. This problem only occurs with those files.  
When playing the original file, I get this error:
```
Python(4833,0x7fffe77c73c0) malloc: *** error for object 0x104e2ce80: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
```
And if I convert the file to OGG using Audacity, the sound comes out at a much lower pitch than it should—but only in pygame.

```python
import pygame
pygame.init()
sound = pygame.mixer.Sound('test.wav')
sound.play()
while True: pass
```

**Comment by illume** [comment]
Hello. It seems you're on some sort of linux? Can you please let me know which one, and how you installed pygame?

I wonder if it is related to a 'manylinux' build obtained by pip. If so, can you please try compiling from source, or use the distro pygame (from apt-get or such)?

**Comment by useless-programs** [comment]
I am actually running macOS Sierra, and I installed pygame via pip3.

**Comment by illume** [comment]
This is probably a sdl_mixer issue. 

It would be useful to test:
- [ ] playing these wav files just using SDL1 without pygame
- [ ] playing these wav files just using SDL2 without pygame

**Comment by illume** [comment]
Yeah, pretty sure this issue is fixed in the unreleased SDL1, and in SDL2. There's more details in this issue now: https://github.com/pygame/pygame/issues/406 So closing this issue. The next pygame release will include the fix.

**Comment by mdorier** [comment]
Has this issue been corrected in SDL? I'm on Mac OS X with pygame 1.9.6, using both `pygame.mixer.Sound` and `pygame.mixer.music`, and both mp3 and ogg are played with a lower pitch than they should.
